lib/commands/publish.ts

```diff
@@ -0,0 +1,55 @@
+﻿///<reference path="../.d.ts"/>
+"use strict";
+
+import commandParams = require("../common/command-params");
+
+export class PublishListCommand implements ICommand {
+    constructor(private $injector: IInjector,
+        private $publishService: IPublishService) { }
+
+    allowedParameters: ICommandParameter[] = [new commandParams.StringCommandParameter(this.$injector), 
+        new commandParams.StringCommandParameter(this.$injector), new commandParams.StringCommandParameter(this.$injector)];
+
+    execute(args: string[]): IFuture<void> {
+        if (args.length === 0) {
```

Comment:
This if looks very peculiar. Can we just remove it and make publish|list default command?